### PR TITLE
Misc - Changes / Fixes

### PR DESCRIPTION
**Pull request by artdude543**
### Changes
- Fixed: Various spelling mistakes and a few URI's being wrong (Extra forward slashes which were not needed)
- Fixed: Some formatting on the descriptions and added spaces where appropriate.
- Updated: Endpoint responses (examples) As they were severely outdated and some developers were getting confused on what data we meant to be returned.
- Changed: Removed some parameters which are not used for the endpoint and tweaked some wording on the descriptions.

### Screenshots:
  - Achievements:
    - All: http://i.imgur.com/LkcdQgu.png
    - User: http://i.imgur.com/daF20A6.png
  - Chats:
    - Get: http://i.imgur.com/GijSq43.png

I can get more screenshots if you want of the other things I've changed / fixed but I gave some examples so you can see the changes ^_^ I will go over the rest of the endpoints when I get more spare time. But here is say batch 1 of a few commits maybe? ;)

### CODE REVIEW

File: src/api/docs/channel/types.js [hunk 1]
```diff
@@ -49,24 +49,52 @@ class SuccesfulResult extends Example {
...
+            225
+          ]
+        },
+        id: 1713,
+        type: "channel:thumbnail",
+        relid: 129,
+        url: "https://s3.amazonaws.com/uploads.beam.pro/thumbnails/129.jpg",
+        store: "s3",
+        remotePath: "thumbnails/129.jpg",
+        createdAt: "2015-02-19 19:12:02",
```
Comment (ttaylorr) [inline src/api/docs/channel/types.js]: How about we use  `createdAt: new Date().toISOString()` here instead?

**Comment by ttaylorr** [comment]
Thank you so much for this!  Really appreciate your work here.  Aside from that one stylistic comment, looks good to me.

@connor4312 any API changes we missed here?

File: src/api/docs/channel/types.js (hunk 1 above)
Comment (artdude543) [inline src/api/docs/channel/types.js]: Ahh damn missed a few xD Will fix!

Comment (connor4312) [inline src/api/docs/channel/types.js]: Meh. If we were to be pedantic, might have a date generator class in the event that the format ever changes. Whichever.

**Comment by connor4312** [comment]
:eyes: good

**Comment by ttaylorr** [comment]
Thanks again!

**Pull request merged**